### PR TITLE
AB#78323-Implement new dashboard filter style

### PR DESCRIPTION
**Pull request by viniciusriquieri1**
# Description

Change the behavior of the filter drawer so that it pushes away the page content instead of overlapping it.

## Useful links
https://dev.azure.com/WHOHQ/EMSSAFE/_workitems/edit/78323

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Improvement (refactor or addition to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Activating filtering and opening the filter drawer it shows the expected behavior.

## Screenshots

Screenshot from 2023-11-02 15-22-11

image

# Checklist:

( * == Mandatory ) 

- [x] * I have set myself as assignee of the pull request
- [x] * My code follows the style guidelines of this project
- [x] * Linting does not generate new warnings
- [x] * I have performed a self-review of my own code
- [x] * I have put the ticket for review, adding the oort-frontend team to the list of reviewers
- [x] * I have commented my code, particularly in hard-to-understand areas
- [x] * I have put JSDoc comment in all required places
- [x] * My changes generate no new warnings
- [x] * I have included screenshots describing my changes if relevant
- [x] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by viniciusriquieri1** [comment]
When the drawer is at the top, it "pushes away" the screen's content, which works well. However, if its position is different, I haven't been able to apply this behavior yet.

**Comment by viniciusriquieri1** [comment]
Still pending the implementation for red dot badge and disabling the icon

**Comment by AntoineRelief** [comment]
@viniciusriquieri1 
when you create a PR that affects the UI, it is important to reflect the changes in the PR screenshots section
there, I can still see that the old screenshots are used
please update them

File: libs/shared/src/lib/components/layout/layout.component.html [hunk 1]
```diff
@@ -42,6 +42,16 @@ <h1 class="text-xl mb-0 whitespace-nowrap text-clip font-medium">
         {{ 'common.platform.backOffice' | translate }}
       </h1>
     </ng-template>
+    <ui-button
+      [isIcon]="true"
+      [size]="'large'"
```
Comment (AntoineRelief) [inline libs/shared/src/lib/components/layout/layout.component.html:47]: @viniciusriquieri1 
when an input is a string you don't need to write it like that:
[input]="'...'"
just do:
input="..."

**Pull request closed without merging**